--- /usr/share/themes/elementary/gtk-2.0/_gtkrc.bak	2014-11-09 21:33:08.962809958 +0100
+++ /usr/share/themes/elementary/gtk-2.0/gtkrc	2014-11-09 21:34:17.850811530 +0100
@@ -68,7 +68,7 @@
 	GtkScrollbar		::has-backward-stepper              = 0  
 	GtkScrollbar		::has-forward-stepper               = 0
 	GtkScrollbar		::min-slider-length                 = 30
-	GtkScrollbar		::slider-width                      = 6
+	GtkScrollbar		::slider-width                      = 10
 	GtkScrollbar		::trough-border                     = 1
 
 	GtkScrolledWindow	::scrollbar-spacing                 = 0
